tests/integration/targets/launchd/vars/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+
```

Comment:
```suggestion
```
it's also unnecessary afaik

Reply:
I am using that since `ansible-galaxy init` is generating all yaml files with the `---` on the top.

Reply:
I don't think we have a standard for this yet. Some people use it, some don't. I don't mind keeping it (I'd probably even be in favor of adding it to every `.yaml` file).

Reply:
it's a suggestion, imo, any extra stuff (this is completely unnecessary and useless, isn't it?) makes a visual noise.
if you disagree, mark this conversation as resolved

Reply:
I don't have a strong opinion, these are just some dashes so won't fight much for it. I will do whatever the majority of you wants ;)

Reply:
i'd prefer to fight:) kidding;) Ok, you and @felixfontein are for it, so feel free to leave this as it is, but hope once you both change your mind about visual noises in source code:)

Reply:
@Andersson007 I wouldn't consider this as visual noise, it's the default sign for "This is a YAML file" (when ignoring the file extension). But yeah, this is a style question, and opinions on this definitely differ :)